diagram.go

```diff
@@ -7,9 +7,9 @@ import (
 	"github.com/golang/freetype"
 	"github.com/golang/freetype/truetype"
 
-	"github.com/akavel/ditaa/embd"
-	"github.com/akavel/ditaa/fontmeasure"
-	"github.com/akavel/ditaa/graphical"
+	"./embd"
+	"./fontmeasure"
+	"./graphical"
```

Comment:
Yeah I read about this when googling how to solve this. However from what I understand relative imports are only a no-go when it is about libraries. In this case it seems to be simply different parts of the same program. None of them are useful on it's own (otherwise they would be in a different repository anyway). Eg. I left the reference to akavel/polyclip-go as is, since it is an external dependency.

To me the advantages of relative paths here are clear, the disadvantages not. But as I said - I'm not a go programmer.